assets/styles.css

```diff
@@ -0,0 +1,930 @@
+button {
+  border: none;
+  font-weight: bold;
+}
+
+* {
+  z-index: 2;
+}
+
+a {
+  z-index: 10000;
+  color: inherit;
+}
+
+body {
+  z-index: 1;
+}
+
+.img-fluid {
+  width: 100%;
+}
+
+a:hover {
+  color: inherit;
+  text-decoration: none;
+}
+
+main {
+  padding: 10px 2.5% !important;
+}
+
+.hamburguer_menu {
+  background-color: transparent;
+  color: rgb(230, 230, 230);
+  display: none;
+}
+
+.header_logo {
+  background-color: red;
+  padding: 16px 10px;
+}
+
+.newsweek_logo {
+  color: white;
+  fill: white;
+}
+
+.newsweek_logo_red {
+  color: white;
+  fill: red;
+}
+
+.buttons_signin {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.gray_white_block {
+  background: gray;
+  color: white;
+}
+
+.white_red_block {
+  background: white;
+  color: red;
+}
+
+.red_white_block {
+  background: red;
+  color: white;
+}
+
+.gray_red_block {
+  background-color: aliceblue;
+  color: red;
+}
+
+.text_date {
+  font-size: 14px;
+  font-weight: 100;
+  color: white;
+}
+
+.black_red_link {
+  color: black;
+}
+
+.black_red_link:hover {
+  color: red;
+  text-decoration: none;
```

Comment:
Use bootstrap class

Reply:
Hi! believe that changing the color text to red and removing the underline decoration from an anchor are project-specific functionalities, therefore, I can't use bootstrap classes for this .